src/macros.rs

```diff
@@ -8,7 +8,9 @@ macro_rules! wait_for_ctrl_c {
             Ok(()) => {
                 println!("\nShutting down...");
                 // Send shutdown messages
-                $($actor.send(StandardMessage::Shutdown);)*
+                $(let _ = $actor.try_send(Message::<StandardPayload>::new(
+                    777,
```

Comment:
Maybe add a `//TODO` noting to comeback to this when we find a solution for this.